src/algebra/big_operators/finprod.lean

```diff
@@ -430,6 +430,15 @@ begin
   simp [mul_support_mul]
 end
 
+@[to_additive] lemma finprod_pow (hf : (mul_support f).finite) (n : ℕ) :
```

Comment:
If you move this lemma after line 520 the proof is simply `(pow_monoid_hom n).map_finprod hf`

Reply:
I think you should add the lemma anyway. It is more convenient to search for `finprod_pow`